ecosystem/index.html

```diff
@@ -173,6 +173,14 @@ <h2>Functional Test Services</h2>
             </div>
             <div class="row-description row-component">Selenium cloud testing in 300+ Real Desktop and Mobile Browsers.</div>
         </div>
+	<div class="row">
```

Comment:
Thanks @muditlambda for the PR, we can accept it but I would say that we should keep the order that we already have (which I don't know where it came from, I guess from historical reasons). Could you please move the addition to the end of the list?